EIPS/eip-9999.md

```diff
@@ -0,0 +1,84 @@
+---
+eip: 
```

Comment:
```suggestion
eip: 7716
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.